libraries/botbuilder-core/src/botState.ts

```diff
@@ -119,7 +112,7 @@ export class BotState implements PropertyManager {
      * @param force (Optional) if `true` the state will always be written out regardless of its change state. Defaults to `false`.
      */
     public saveChanges(context: TurnContext, force: boolean = false): Promise<void> {
-        let cached: any = context.turnState.get(this.stateKey) as CachedBotState;
+        let cached: CachedBotState = context.turnState.get(this.stateKey);
```

Comment:
technically this is a code change.  are we ok with this?

Reply:
same as above

---
In reply to: [219688303](https://github.com/Microsoft/botbuilder-js/pull/477#discussion_r219688303) [](ancestors = 219688303)